src/contentscript/overlay/root/common/constans.ts

```diff
@@ -0,0 +1,8 @@
+export const regExpIndexNearTestnet = new RegExp(
```

Comment:
* Misprint: `constans.ts` => `constants.ts`
* The similar code is located here: https://github.com/dapplets/dapplet-extension/blob/ce6ac3cb0b74aeb775070fa0dd186d7dba864144/src/common/helpers.ts#L49 
It'd be great to reuse this function and move `RegExp`s there